nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -642,6 +642,10 @@ in
       }
     ];
 
+    systemd.tmpfiles.rules = [
+      "d ${cfg.stateDir} 0750 ${cfg.user} ${cfg.group} - -"
```

Comment:
You should quote stateDir:
"d '${cfg.stateDir}' 0750 ${cfg.user} ${cfg.group} - -"

Reply:
Update, thanks.